File: apollo/src/main/java/io/muun/apollo/domain/errors/PasswordTooShortError.java
package io.muun.apollo.domain.errors;


import io.muun.apollo.external.UserFacingErrorMessages;

public class PasswordTooShortError extends UserFacingError {

    public PasswordTooShortError() {
        super(UserFacingErrorMessages.INSTANCE.passwordTooShort());
    }
}

File: android/apolloui/src/main/java/io/muun/apollo/domain/errors/passwd/PasswordTooShortError.kt
package io.muun.apollo.domain.errors.passwd


import io.muun.apollo.data.external.UserFacingErrorMessages
import io.muun.apollo.domain.errors.ErrorClassification
import io.muun.apollo.domain.errors.UserFacingError

class PasswordTooShortError : UserFacingError(
    UserFacingErrorMessages.INSTANCE.passwordTooShort()
) {
    override val classification = ErrorClassification.EXPECTED
}
